src/lib/rollback-manager.ts

```
@@ -0,0 +1,350 @@
+   * Reset a specific circuit
+   */
+  static reset(featureName: string): void {
+    this.failures.delete(featureName);
+    this.lastFailureTime.delete(featureName);
+    this.circuitState.delete(featureName);
+    console.log(`[CircuitBreaker] Reset circuit for ${featureName}`);
+  }
+}
+
+// Export singleton instance
+export const rollbackManager = RollbackManager.getInstance();
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)